src/components/carousel/index.js

```diff
@@ -441,14 +445,15 @@ export default class Carousel extends BaseComponent {
   }
 
   renderCarousel() {
-    const {containerStyle, ...others} = this.getThemeProps();
+    const {containerStyle, animate, ...others} = this.getThemeProps();
     const {initialOffset} = this.state;
     const scrollContainerStyle = this.shouldUsePageWidth() ? {paddingRight: this.getItemSpacings(this.getThemeProps())} : undefined;
     const snapToOffsets = this.getSnapToOffsets();
     const marginBottom = Math.max(0, this.getContainerPaddingVertical() - 16);
+    const Container = animate ? Animated.View : View;
 
     return (
-      <View style={[{marginBottom}, containerStyle]} onLayout={this.onContainerLayout}>
+      <Container style={[{marginBottom}, containerStyle]} onLayout={this.onContainerLayout}>
```

Comment:
Should I always use Animated.View and remove the. new prop?

Reply:
What is this change for exactly?

Reply:
This is for our private #1065